app/models/person.rb

```diff
@@ -17,6 +17,7 @@ class Person < ActiveRecord::Base
   validates :surname, presence: true
   validates :email, presence: true
   validates :discipline, presence: true
+  validates :competence, presence: true
```

Comment:
Usunąłbym to wymaganie. Kompetencje nie muszą być określone, a powodują, że trudniej pisze się testy